packages/uniorg-parse/src/parser.ts

```diff
@@ -999,9 +1050,10 @@ class Parser {
   }
 
   private parseClock(): Clock {
+    const begin = this.r.offset();
     this.r.advance(this.r.forceMatch(/^[ \t]*CLOCK:[ \t]*/));
     const value = this.parseTimestamp();
-
+    const end = this.r.offset();
```

Comment:
❗🐛 the `end` does not account for advances on the next two lines. Need to move this line just before `this.parseEmptyLines()`